conans/client/edited.py

```diff
@@ -0,0 +1,44 @@
+import json
+import os
+from os.path import join, normpath
+
+from conans.model.ref import ConanFileReference
+from conans.util.files import load, save
+
+
+EDITED_PACKAGES = 'edited_packages'
```

Comment:
If this is a file, name according to it:

```suggestion
EDITED_PACKAGES_FILE = 'edited_packages'
```